common/reviews/api/module-minifier-plugin.api.md

```diff
@@ -205,14 +211,6 @@ export const STAGE_AFTER: 100;
 // @public
 export const STAGE_BEFORE: -100;
 
-// @public
-export class SynchronousMinifier implements IModuleMinifier {
```

Comment:
This is a breaking change.

Reply:
David and I discussed this earlier, but I think a breaking change is fair here. For one, it no longer runs synchronously so labeling it as such is a bit misleading. Also, it's still a `0.x.x` package so the minor bump behaves like a major bump in terms of `~` and `^` versions in `package.json`.

Reply:
Sounds reasonable, then.